src/StoreApi/Routes/V1/OrderUpdateCustomer.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Routes\V1;
+
+use Automattic\WooCommerce\StoreApi\SchemaController;
+use Automattic\WooCommerce\StoreApi\Schemas\v1\AbstractSchema;
+use Automattic\WooCommerce\StoreApi\Utilities\OrderAuthorizationTrait;
+use Automattic\WooCommerce\StoreApi\Utilities\OrderController;
+use Automattic\WooCommerce\StoreApi\Utilities\UpdateCustomerTrait;
+
+/**
+ * OrderUpdateCustomer class.
+ *
+ * Updates the customer billing and shipping addresses and returns an updated order--taxes should be recalculated.
```

Comment:
It's a little unclear what the comment means exactly, is it saying the taxes _should_ be recalculated separately after using this, or that they will be recalculated as a result of running this? Please could you try to make that a little clearer? 

If the latter, I'm curious about the word `should`, that implies there's a case where they might not, is that true?

Reply:
That would be super helpful if you could update both, it adds a little noise to the PR but I think it's worth it.

Reply:
Updated in [8934758](https://github.com/woocommerce/woocommerce-blocks/pull/9743/commits/8934758e5be61269c880ec677f893cb7b1cfa9e7).